### ISSUE TITLE
Please include shortcut options for left side modules in darkroom

### ISSUE THREAD

**Issue by StefanDieke** [darktable-org/darktable#4126]
In my workflow in the darkroom I'm using keyboard shortcuts quite a lot to switch between modules or switch them on or off. After getting used to it, it speeds up the process significantly and now it feels really uncomfortable not to have shortcuts for modules which are often used.

Unfortunately there is no option to assign shortcuts for those modules on the left side, e.g. the stack, image information, mask manager and so on. 

The three I mentioned may take up a lot of space as they grow during the editing process. So that switching from one to another may require bringing the mousepointer to the scrollbar, scroll the mousewheel and a ckick on the module you are interested in.

Keyboard shortcuts could speed this up significantly. 

So it would be nice, if you could assign the same options for setting up keyboard shortcuts for the modules in the left column as for those in the right column. For my workflow options to unfold / fold the modules or to focus (scroll) them would be sufficiant but I think it would be generally be more helpfull if all modules would offer the same options for keyboard assignment, no matter if they seem to make sense for the individual module or not. In my experience others have come up with clever applications of options, which I couldn't even imagine.

**Comment by junkyardsparkle** [comment]
What version are you using? I see shortcuts available for all of these in current master (mask manager is under "image operations", others are under "modules").

**Comment by StefanDieke** [comment]
Thank you for the pointers!

Now I found the keyboard-setup for the mask manager and for tags. But it seems, as if it doesn't work. I asigned 'X' to 'show module' for the 'mask manager'. Hitting x doesn't do anything. It doesn't matter if masks already exist or not. 
The same key works fine if assigned to 'show module' for e.g. 'colour ballance'.

But I can't find the duplicate manager at all in shortcuts.
And 'image information', 'color picker' and 'stack' don't have options for 'show' and 'activate'.

I'm on 3.1.0+342~g112559639
